test/clients/azure/client.go

```diff
@@ -22,9 +34,26 @@ func NewClientFromEnvironment() (*Client, error) {
 
 	subscriptionID := os.Getenv("AZURE_SUBSCRIPTION_ID")
 
+	var rpc externalapi.OpenShiftManagedClustersClient
+	focus := []byte(config.GinkgoConfig.FocusString)
+	switch {
+	case fakeRe.Match(focus):
+		fmt.Println("configuring the fake resource provider")
+		rpUrl := "http://" + fakerp.LocalHttpAddr
+		rpc = externalapi.NewOpenShiftManagedClustersClientWithBaseURI(rpUrl, subscriptionID)
+	case realRe.Match(focus):
```

Comment:
I think we need an additional focus here for [Admin] tests to use the the [admin](https://github.com/openshift/openshift-azure/blob/2ec69b94f480a12d231cbec6fe5b4a461d1f41f6/pkg/api/admin/api/client.go#L89) client.